api/tests/unit/domain/usecases/disable-certification-center-membership_test.js

```diff
@@ -0,0 +1,33 @@
+const { expect, sinon } = require('../../../test-helper');
+const { disableCertificationCenterMembership } = require('../../../../lib/domain/usecases');
+const certificationCenterMembershipRepository = require('../../../../lib/infrastructure/repositories/certification-center-membership-repository');
+
+describe('Unit | UseCase | disable-certification-center-membership', function () {
+  const now = new Date('2019-01-01T05:06:07Z');
+  let clock;
+
+  beforeEach(function () {
+    clock = sinon.useFakeTimers(now);
+  });
+
+  afterEach(function () {
+    clock.restore();
+  });
+
+  it('should disable certification center membership', async function () {
+    // given
+    const certificationCenterMembershipId = 100;
+    sinon.stub(certificationCenterMembershipRepository, 'disableById');
+
+    // when
+    await disableCertificationCenterMembership({
+      certificationCenterMembershipId,
+      certificationCenterMembershipRepository,
+    });
+
+    // then
+    expect(certificationCenterMembershipRepository.disableById).to.has.been.calledWithExactly({
```

Comment:
Ca semble etre OK mais ca sonne pas anglais :D
```suggestion
    expect(certificationCenterMembershipRepository.disableById).to.have.been.calledWithExactly({
```